<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnlineDialog</class>
 <widget class="QDialog" name="OnlineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>810</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pb_icon">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/tb/fgroup.png</normaloff>:/tb/fgroup.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>未登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_login">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>登录/注册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_rtmpUrl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>461</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>输入rtmp地址</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_rtmpPlay">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>播放rtmp链接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_tvShow">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>地方台直播</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_video">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>推荐影视</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_fresh">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新影视</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_upload">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>上传视频</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_uploadHistory">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>上传历史</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="sw_page">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>811</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QPushButton" name="pb_tv1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/tb/123456.jpeg</normaloff>:/tb/123456.jpeg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_tv2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/tb/456789.jpeg</normaloff>:/tb/456789.jpeg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="pb_play1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pb_play2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pb_play3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>0</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pb_play4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>0</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pb_play5">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>0</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pb_play10">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>130</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pb_play8">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pb_play9">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>130</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pb_play7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pb_play6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pb_play15">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>260</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pb_play13">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>260</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pb_play14">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>260</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pb_play11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pb_play12">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>260</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
